src/components/Modal/SettingsModal.tsx

```diff
@@ -182,28 +187,41 @@ const DeveloperButton = (props: { t: i18nTranslate }) => {
           ? isTestnets
             ? t?.('enableMainnets') || 'Enable mainnets'
             : t?.('enableTestnets') || 'Enable testnets'
-          : `(${5 - count})`
+          : `(${t?.('clickMoreTimes', { n: (5 - count).toString() }) || 5 - count})`
```

Comment:
Will this end up rendering the string "Click 1 more times"?

Reply:
It's currently set to the following on Locize:

![image](https://user-images.githubusercontent.com/22108522/200943288-f366631c-e18a-4e78-a31b-fb0a15d19e94.png)

So, technically... yes? But it didn't seem too weird to include an `(s)` and leave it at a simpler implementation.

Reply:
Sounds good 👍
For future reference the package we use supports plurals so you can use that next time! It's super easy.